setup_data_scheme.md

```diff
@@ -486,3 +626,35 @@ example.prototype.validatorSchema = {
   }
 }
 ```
+## Data Loader
+For `readOne<model>` API, data loader would be used by default. It means that we can collect IDs, then fetch multiple records within one query. Hence, the read performance could be improved in Zendro.
```

Comment:
When reading a record by its id, by default Zendro uses a [data loader](https://github.com/graphql/dataloader) to improve read performance. It does so by bundling IDs to be fetched and request those in one composite query.